doc/adr/0014-synchronous_peer_discovery.md

```diff
@@ -0,0 +1,73 @@
+# 14. Synchronous Peer Discovery
```

Comment:
in general, it should be the very next number. I know that 0013 is under review but we cannot merge 0014 before 0013. Hence, I'd set 0013 here, and we can merge it after agreement. If this ADR will be the first one merged, we'll update the number of the second one (and vice versa).

Reply:
Changed.